tests/components/api/test_init.py

```diff
@@ -300,12 +301,13 @@ async def test_api_get_services(
     """Test if we can get a dict describing current services."""
     resp = await mock_api_client.get(const.URL_API_SERVICES)
     data = await resp.json()
-    local_services = hass.services.async_services()
 
-    for serv_domain in data:
-        local = local_services.pop(serv_domain["domain"])
+    descriptions = await async_get_all_descriptions(hass)
```

Comment:
These changes don't seem related to the newly added service.
Should probably best be split off into a separate PR.